src/us_reverse_geo/SmartyResponse.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * A candidate is a possible match for an address that was submitted.<br>
```

Comment:
Is this comment description up to date? It doesn't seem to match the class defined in this file